itest/rust/src/builtin_tests/script/script_instance_tests.rs

```diff
@@ -38,7 +38,9 @@ impl IScriptExtension for TestScript {
 struct TestScriptInstance {
     /// A field to store the value of the `script_property_b` during tests.
     script_property_b: bool,
+    #[allow(clippy::arc_with_non_send_sync)]
     prop_list: Vec<PropertyInfo>,
+    #[allow(clippy::arc_with_non_send_sync)]
     method_list: Vec<MethodInfo>,
```

Comment:
```suggestion
    prop_list: Vec<PropertyInfo>,
    method_list: Vec<MethodInfo>,
```

These `allow`s shouldn't be needed here right?

Reply:
thanks.